chart/templates/webserver/webserver-networkpolicy.yaml

```diff
@@ -39,13 +39,13 @@ spec:
       component: webserver
       release: {{ .Release.Name }}
   policyTypes:
-  - Ingress
-  ingress:
-  - from:
+    - Ingress
 {{- if .Values.webserver.extraNetworkPolicies }}
-{{ toYaml .Values.webserver.extraNetworkPolicies | indent 4 }}
+  ingress:
+    - from:
+{{ toYaml .Values.webserver.extraNetworkPolicies | indent 6 }}
```

Comment:
I don't understand how to use option `extraNetworkPolicies` in NetworkPolicy.  I tried a similar policy in helm / charts and couldn't find anything similar. It seems to me that we should think a little more about what these policies should do.
https://github.com/helm/charts/search?p=2&q=NetworkPolicy

Reply:
I am not that familiar with networkPolicies so no real opinion on this.

There seems to have been a discussion around this on the helm/chart repo: https://github.com/helm/charts/issues/4067
Also those `extraNetworkPolicies` don't seem to have ever existed on astronomer's chart (except for pgbouncer) ; so maybe we can ask the ones who participated in https://github.com/apache/airflow/pull/8777 ?

Reply:
@thesuperzapper  Do you have any experience with this? Can you help us please?

Reply:
@mik-laj I don't know what the creator of this specific PR was trying to achieve, but from context, its just a way for users to specify things under `from` using a list value, for example:
```yaml
webserver:
  extraNetworkPolicies:
    - ipBlock:
        cidr: 172.17.0.0/16
        except:
          - 172.17.1.0/24
    - namespaceSelector:
        matchLabels:
          project: myproject
    - podSelector:
        matchLabels:
          role: frontend
```

You can read about NetworkPolicy here: https://kubernetes.io/docs/concepts/services-networking/network-policies/

But this is just another case where there needs to be significant documentation updates for this chart before it's ready for normal users.